accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/impl/AcInstallationServiceImpl.java

```diff
@@ -744,111 +754,175 @@ public String purgeACLs(String path) {
     @Override
     public String purgeAuthorizablesFromConfig() {
         List<String> resultMessages = new ArrayList<String>();
-        for(String configRootPath: configurationRootPaths) {
+        for (String configRootPath : configurationRootPaths) {
             LOG.info("Purging authorizables for root path {}", configRootPath);
             resultMessages.add(purgeAuthorizablesFromConfig(configRootPath));
         }
         return StringUtils.join(resultMessages, "\n");
     }
-    
+
     @Override
     public String purgeAuthorizablesFromConfig(String configRootPath) {
         Session session = null;
-        String message = "";
         try {
             session = repository.loginService(null, null);
 
-            Set<String> authorizabesFromConfigurations = getAllAuthorizablesFromConfig(session, configRootPath);
-            message = purgeAuthorizables(authorizabesFromConfigurations, session);
             PersistableInstallationLogger installLog = new PersistableInstallationLogger();
-            installLog.addMessage(LOG, "purge method: purgAuthorizablesFromConfig()");
-            installLog.addMessage(LOG, message);
+            installLog.addMessage(LOG, "*** Purging AC Tool configuraiton " + configRootPath + "...");
+
+            Map<String, String> newestConfigurations = configFilesRetriever.getConfigFileContentFromNode(configRootPath, session);
+            AcConfiguration acConfiguration = configurationMerger.getMergedConfigurations(newestConfigurations, installLog, configReader,
+                    session);
+
+            installLog.addMessage(LOG, "Purging ACLs...");
+            long startAclPurge = System.currentTimeMillis();
+            // removing the ace config section will clear all
+            acConfiguration.getAceConfig().clear();
```

Comment:
Runs a regular installation without ACE configs, therefore `umanagedAcePathsRegex` is respected and for fragment mentioned in #535 it will only remove the ACEs underneath path `/apps/myproj/login` (and not the ootb ACEs for everyone)